src/to_vec.jl

```diff
@@ -21,6 +21,15 @@ end
 # Base case -- if x is already a Vector{<:Real} there's no conversion necessary.
 to_vec(x::Vector{<:Real}) = (x, identity)
 
+# get around the constructors and make the type directly
+@generated function _force_construct(T, args...)
+    return if VERSION >= v"1.3"
```

Comment:
should the `if VERSION` be inside or outside of the function? (i've no idea if it makes a difference here, but curious if it does)

Reply:
Putting it on the outside will be easier on the compiler i think.